perun-core/src/main/resources/postgresChangelog.txt

```diff
@@ -6,6 +6,10 @@
 -- Directly under version number should be version commands. They will be executed in the order they are written here.
 -- Comments are prefixed with -- and can be written only between version blocks, that means not in the lines with commands. They have to be at the start of the line.
 
+3.2.11
+ALTER TABLE vos DROP constraint vo_u unique (name);
```

Comment:
I think that if you drop the constraint you should just define a name without the rest of the definition part.
```suggestion
ALTER TABLE vos DROP constraint vo_u;
```